### PR TITLE
LPS-165907 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for the [LPS-165907](https://github.com/sontruongces/liferay-portal/pull/51), the original has passed my review at sontruongces#51. Please help me to review and give feedback.

Thanks.
cc: @TaiPhamLR

### CODE REVIEW

File: modules/apps/portal-instances/portal-instances-web/src/main/java/com/liferay/portal/instances/web/internal/portlet/action/AddInstanceMVCActionCommand.java [hunk 1]
```diff
@@ -85,6 +87,11 @@ else if (exception instanceof CompanyWebIdException) {
 			jsonObject.put(
 				"error",
 				_language.get(actionRequest.getLocale(), errorMessage));
+
+			SessionMessages.add(
+				actionRequest,
+				_portal.getPortletId(actionRequest) +
+					SessionMessages.KEY_SUFFIX_HIDE_DEFAULT_SUCCESS_MESSAGE);
```
Comment (pei-jung) [inline modules/apps/portal-instances/portal-instances-web/src/main/java/com/liferay/portal/instances/web/internal/portlet/action/AddInstanceMVCActionCommand.java:94]: This is already defined in `hideDefaultSuccessMessage` in the `BaseMVCActionCommand`. Simply call the `hideDefaultSuccessMessage` here.

Comment (TaiPhamLR) [reply modules/apps/portal-instances/portal-instances-web/src/main/java/com/liferay/portal/instances/web/internal/portlet/action/AddInstanceMVCActionCommand.java]: Hi @pei-jung, thanks for your review. I fixed this.

**Review by drewbrokke — APPROVED** [review]
Looks good, thanks for updating the pull with the feedback, @TaiPhamLR!

**Comment by patriciajeanperez** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/125387

**Pull request closed without merging**